src/Utils/GeneralComponents/useCountdown.js

```diff
@@ -0,0 +1,30 @@
+import { useState, useEffect } from "react";
+import moment from "moment";
+import "moment-duration-format";
+export default function useCountdown(startTime, endTime) {
+  const [duration, setDuration] = useState(
+    moment.duration(endTime - startTime)
```

Comment:
perhaps I'm misunderstanding, but souldn't `duration` be calculated as `endTime - currentTime`?
I believe that the fact that the `startTime` is generated as the current time cover up for a logical mistake here.

Reply:
the start time might be the current time but in the future (when the countdown started) it is not the current time anymore. that is why i called it start time.